sdk/tools/azure-sdk-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,142 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>jar</packaging>
+
+    <name>${artifactId}</name>
+
+    <properties>
+        <maven.compiler.target>${javaVersion}</maven.compiler.target>
+        <maven.compiler.source>${javaVersion}</maven.compiler.source>
+        <bom.version>1.0.2</bom.version>
+        <graalvm.version>21.2.0</graalvm.version>
+#if( ${junitVersion} == '4')
+        <junit4.version>4.12</junit4.version>
+#end
+#if( ${junitVersion} == '5')
+        <junit5.version>5.7.2</junit5.version>
+#end
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.azure</groupId>
+                <artifactId>azure-sdk-bom</artifactId>
+                <version>${bom.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+
+#set($libraries = ${azureLibraries})
+#set($dependencies = $libraries.split(","))
+#foreach($dependency in $dependencies)
```

Comment:
What happens if the user picks an Azure SDK that doesn't exist in the BOM

Reply:
The generated pom.xml will have whatever was specified by the user and will show up as an error.  For e.g., when user enters `azure-identity, azure-foo-service`, the dependency section will be as shown below:

```xml
   <dependency>
      <groupId>com.azure</groupId>
      <artifactId>azure-identity</artifactId>
    </dependency>
    <dependency>
      <groupId>com.azure</groupId>
      <artifactId>azure-foo-service</artifactId> <!-- this will be an invalid dependency and will cause errors -->
    </dependency>
```

Reply:
Should we have the logic to add dependencies not in BOM to have the <version> tag instead?

Reply:
To know which dependency is not in BOM, this archetype has to first download the BOM, read and check if a particular dependency is missing and then add the version tag. It can be added as an enhancement if that is required but with regular BOM releases, it may not be necessary. If we want to do it, there's a post-generation groovy script that runs where we can add this logic.